### PR TITLE
Vescavors be gone

### PR DESCRIPTION
**Pull request by Shadowy-Ranger**
Adds a toggle to enable removing vescavor models, similar to spiders be gone.  Implements #403

### CODE REVIEW

File: ToyBox/ReadMe.md [hunk 1]
```diff
@@ -28,6 +28,9 @@
 * **Crusade Editor**
   * Reworked UI to match design of other parts of the mod
   * Fixed bug where it was displaying the Exp required for the current rank rather than the next rank
+* (***ShadowRanger***) Bugfix for taking 10 out of combat: corrected missing IsInCombat flag
+* (***ShadowRanger***) Added a vescavors-begone option, similar to spiders-begone, to replace all vescavor models
```
Comment (cabarius) [inline ToyBox/ReadMe.md:32]: If it is convenient, please try to do separate pull requests for separate patches but this is awesome, thanks for the hard work.

**Review by cabarius — APPROVED** [review]

**Review by cabarius — APPROVED** [review]

File: ToyBox/ReadMe.md (hunk 1 above)
Comment (Shadowy-Ranger) [reply ToyBox/ReadMe.md]: Ah; might just be me messing up the git forking / merging.  I believe the PR with the bugfix for taking 10 got pulled in yesterday.
